back/engines/free/email_campaigns/app/models/concerns/email_campaigns/trackable.rb

```diff
@@ -24,26 +24,17 @@ def last_delivery_for_recipient(user)
         &.sent_at
     end
 
-    def extra_mailgun_variables
-      if !@command
-        ErrorReporter.report_msg('No command in extra_mailgun_variables!')
-        @command = {}
-      end
-      if !@command[:delivery_id]
-        ErrorReporter.report_msg('No delivery ID in extra_mailgun_variables!')
-      end
-      { 'cl_delivery_id' => @command[:delivery_id] }
+    def extra_mailgun_variables(command)
+      { 'cl_delivery_id' => command[:delivery_id] }
     end
 
     private
 
     def generate_delivery_id(command)
-      @command = command
-      command[:delivery_id] ||= SecureRandom.uuid
+      command[:delivery_id] = SecureRandom.uuid
     end
 
     def save_delivery(command)
-      ErrorReporter.report_msg('No delivery ID in save_delivery!') if !command[:delivery_id]
```

Comment:
Worth keeping this in, just in case?

Reply:
I put it back in `extra_mailgun_variables`, where the issue was occurring.